src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/casenotes/CaseNotesReconciliationService.kt

```diff
@@ -143,6 +128,38 @@ class CaseNotesReconciliationService(
       )
     }.getOrNull()
   }
+
+  private fun transformFromDps(c: CaseNote) = CommonCaseNoteFields(
+    c.text,
+    c.type,
+    c.subType,
+    c.occurrenceDateTime.format(DateTimeFormatter.ISO_DATE_TIME),
+    c.authorUsername,
+    c.amendments.map { a ->
+      CommonAmendmentFields(
+        text = a.additionalNoteText,
+        occurrenceDateTime = a.creationDateTime?.format(DateTimeFormatter.ISO_DATE_TIME),
+        authorUsername = a.authorUserName,
+      )
+    }.toSortedSet(amendmentComparator),
```

Comment:
minor: I think if this was an extension function of `CaseNote`  it would read better; `c.text` becomes `text` and `a.additionalNoteText` would be `it.additionalNoteText` since you wouldn't need either variable